controllers/submariner/submariner_controller.go

```diff
@@ -225,12 +225,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
-	err = updateDaemonSetStatus(ctx, r.config.Client, globalnetDaemonSet, &instance.Status.GlobalnetDaemonSetStatus, request.Namespace)
-	if err != nil {
-		reqLogger.Error(err, "failed to check gateway daemonset containers")
+	// BILLY: Revisit
+	// err = updateDaemonSetStatus(ctx, r.config.Client, globalnetDaemonSet, &instance.Status.GlobalnetDaemonSetStatus, request.Namespace)
+	// if err != nil {
+	// 	reqLogger.Error(err, "failed to check gateway daemonset containers")
 
-		return reconcile.Result{}, err
-	}
+	// 	return reconcile.Result{}, err
+	// }
```

Comment:
Is this still relevant? If so can you make a TODO and describe what needs to be done?

Reply:
@tpantelis Updated comment. See commit https://github.com/submariner-io/submariner-operator/pull/2027/commits/565529a638b7c43c356830f01258bb874b3d4b85